featurebyte/api/user_defined_function.py

```diff
@@ -344,6 +344,29 @@ def create(
         user_defined_function.save()
         return user_defined_function
 
+    @classmethod
+    def get(cls, name: str) -> UserDefinedFunction:
+        """
+        Gets a UserDefinedFunction object by its name.
```

Comment:
Should this use singular `Get` to be consistent with the style of other docstrings?

Reply:
same with FeatureStore 
https://docs.featurebyte.com/0.6/reference/featurebyte.api.feature_store.FeatureStore.get/

Catalog 
https://docs.featurebyte.com/0.6/reference/featurebyte.api.catalog.Catalog.get/

Reply:
Yeah that's what I wanted to say, sorry for miswording. I think either way makes sense, but it's definitely not a blocking issue.